Contuity/Models/People.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  People.swift
+//  Contuity
+//
+//  Created by Lexi Spikerman on 10/2/18.
+//  Copyright © 2018 Generate. All rights reserved.
+//
+
+import Foundation
+
+class People {
+    
+    var id: Int
+    var name: String?
+    var number: String?
```

Comment:
What does number represent again? like a phone number? would it make sense to rename this var to be more clear